### DISCUSSION TITLE
No Quotation marks in some of the Meta tags?!

### DISCUSSION THREAD

**Discussion by gangajamuna** [arthelokyo/astrowind#87] [Q&A]
Hi it came to my attention that many of the meta tags have no quotation marks in the template. For example ``` <meta charset=UTF-8> ```
Shouldn't it be ``` <meta charset="UTF-8"> ```?
Thx

**Comment by vasfvitor** [comment]
Could you link to a file for example?

**Comment by widgeter** [comment]
Hello @gangajamuna 

I just realized that. I don't know if it will be correct that way or it will be an Astro error. I'll investigate.

`view-source:https://astrowind.vercel.app/`

```html
<!DOCTYPE html><html class=2xl:text-[20px] dir=ltr lang=en><head><meta charset=UTF-8><meta content="width=device-width,initial-scale=1" name=viewport> ...
```

**Comment by widgeter** [comment]
Hi @gangajamuna 

I already found the problem, and it's really not a problem. It is considered that in HTML there are some attribute / value combinations in which quotes can be removed. For that reason the `astro-compress` integration making use of `html-minifier-terser` by default is suppressing them.

https://github.com/terser/html-minifier-terser#options-quick-reference

However, it is preferable and recommended that quotes are always used, besides that aesthetically it looks weird. For that reason I changed the `astro-compress` settings in `astro.config.js` so that they are not removed.

https://github.com/onwidget/astrowind/commit/60d9de74f4e4e8fbfdafa089b55fd4c091b47997

Thank you for detecting this detail.

**Reply by gangajamuna** [reply]
Ahh compression, I wondered where it came from! - Thank you so much!
